etc/init.d/zfs.in

```diff
@@ -0,0 +1,74 @@
+# ZoL userland configuration.
+
+# Run `zfs mount -a` during system start?
+ZFS_MOUNT='yes'
+
+# Run `zfs unmount -a` during system stop?
+ZFS_UNMOUNT='yes'
+
+# Run `zfs share -a` during system start?
+# nb: The shareiscsi, sharenfs, and sharesmb dataset properties.
+ZFS_SHARE='yes'
```

Comment:
> @FranzUrbo I agree, with one exception: the package maintainer did not call special attention to a significant change in behavior

It doesn't have to. It's a config file� And there WAS no change to your system, dpkg stopped that from happening, as it was supposed to.

> I humbly request that in the future, if such a change is made, that it be included in the NEWS.Debian file

If it's in a config file, no. If it's somewhere I'm _forcing_ this on the user/admin, I'll try.